dev/com.ibm.ws.cdi.weld/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -13,8 +13,8 @@
 #NLS_ENCODING=UNICODE
 #NLS_MESSAGEFORMAT_NONE
 #
-cdi12=Contexts And Dependency Injection (CDI) V1.2
-cdi12.desc=Defines the behavior of the Contexts and Dependency Injection (CDI) v1.2.
+cdi12=Contexts And Dependency Injection V1.2 or newer
+cdi12.desc=Defines the behavior of the Contexts and Dependency Injection (CDI) v1.2 or newer.
```

Comment:
@bcpswope please review these two lines

Reply:
sorry--labeled but did not respond. Looks good.

Reply:
@tevans78 these look good, I only mentioned a change in the copyright statement.